engines/dfc_provider/app/controllers/dfc_provider/api/base_controller.rb

```diff
@@ -26,16 +26,19 @@ def check_user
         head :unauthorized
       end
 
-      def check_enterprise
+      def set_enterprise
         current_enterprise
       end
 
       def current_enterprise
         @current_enterprise ||=
-          if params[enterprise_id_param_name] == 'default'
-            current_user.enterprises.first!
+          case params[enterprise_id_param_name]
+          when 'default'
+            return current_user.enterprises.first!
```

Comment:
I think the `return` is unnecessary here as long as the case statement is the last thing in the method

Reply:
Ah right, correcting this.